scripts/exregional_run_met_genensprod_or_ensemblestat.sh

```diff
@@ -388,31 +388,30 @@ settings="\
   'field_thresholds': '${FIELD_THRESHOLDS:-}'
 "
 
-# Store the settings in a temporary file
+# Render the template to create a METPlus configuration file
 tmpfile=$( $READLINK -f "$(mktemp ./met_plus_settings.XXXXXX.yaml)")
 cat > $tmpfile << EOF
 $settings
 EOF
-#
-# Call the python script to generate the METplus configuration file from
-# the jinja template.
-#
-python3 $USHdir/python_utils/workflow-tools/scripts/templater.py \
-  -c "${tmpfile}" \
+
+uw template render \
```

Comment:
This change was made in several of the ex-scripts. This is how the command line tool can be called to render a template instead of relying on a script on disk.